### PR TITLE
set-up  portofolio

### PR DESCRIPTION
**Pull request by SabaAhmad404**
This is a set-up mobile portfolio website project. In this project, I worked on
1) setting up the linters
2) added Html and CSS files
3) and .gitignore files
kindly review my project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Nyame-Wolf) [inline README.md:55]: [OPTIONAL] Kindly add the licence file MIT.md file here. This currently opens an empty page. The way to do this is described [here](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 
-At the bottom of the readme template you used as well.

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+
+- GitHub: [@SabaAhmad786](https://github.com/SabaAhmad786)
+- LinkedIn: [@saba ahmad](https://www.linkedin.com/in/saba-ahmad-97b938244/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (Nyame-Wolf) [inline README.md]: [OPTIONAL] Kindly add the correct link to the issues page that point to your project.
Screenshot 2022-09-22 at 03 23 50
Its always good practice to do this for all projects you will create.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      <li><img src="pics/Vector1.png" alt="twitter"></li>
+       <li><img src="pics/Linkedin icon.png" alt=""></li>
+      <li><img src="pics/Vector 3.png" alt=""></li>
+       <li><img src="pics/Vector 4.png" alt=""></li>
+       <li><img src="pics/Vector 5.png" alt=""></li>
+        </ul>
+        </div>
+        </div>
+      </section>
+</body>
```
Comment (Nyame-Wolf) [inline index.html]: Kindly note that this section has a [background image](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image), according to the figma design.
Screenshot 2022-09-22 at 16 50 13
It results in the above pointed shapes.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        <div class="container">
+        <a class="logo">Saba Ahmad</a>
+          <nav id="items">
+        <i class="fa fa-bars"></i>
+     </nav>
+     </div>
+</header>
+<section>
+<div class="content">
+    <h1>I'm Saba<br> Glad to see<br> you!</h1>
```
Comment (Nyame-Wolf) [inline index.html]: Kindly note that for the text `Glad to see` and `you` should use a header tag that is  hx `<h1>,<h2>,<h3>`
This is to conform to the semantic correct way  and the screen readers and other technologies will thank you.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.logo:hover {
+  color: darksalmon;
+}
+
+#items {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
```
Comment (Nyame-Wolf) [inline style.css:30]: Great job on getting your project to resemble the figma design. however, this project requires that you follow the figma design's font, colors and also space between the elements
## currently this is your design
Screenshot 2022-09-22 at 16 58 00
ts as well as arrangement.

## this is the figma desidn required.
Screenshot 2022-09-22 at 16 50 13
 

- [ ] Kindly note that its a requirement that the design has the media images below to social media pages.

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @SabaAhmad786  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the same font as Figma design ☑️ 
- Commits are descriptive and meaningful :+1: 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css [hunk 6]
```diff
@@ -0,0 +1,108 @@
+body {
+  background-image: url(pics/shapes.png);
```
Comment (christianbiring1) [inline style.css:2]: [OPTIONAL]
- [ ] Kindly make sure this `image` is set as `background` only for the `first section` and not of the `body` as you can see on the Figma design.

**Review by christianbiring1 — APPROVED** [review]
Hi @SabaAhmad786 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
APPROVED

## Highlights
- [ ] Linter tests are all green
- [ ] Descriptive pull request title and summary
- [ ] You have implemented all the required changes as requested by the previous reviewer.
:rocket: Jeep rolling you're doing a great job :+1:  :mechanical_arm: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**